scripts/pdf_extract.py

```diff
@@ -144,16 +142,18 @@ def add_free_periods(schedule_obj):
     if f in DO_NOT_PARSE: # If the schedule shouldn't be parsed
         print "Skipping"
         continue
+    filepath = "c:\\users\\guberti\\Documents\\Github\\EPSchedule\\schedules\\" + f   #Create the full filepath for the schedule
+    print f
+
+    if os.path.getsize(filepath) <  5000: # If schedule is blank (e.g. is less than 5000 bytes)
+        print "Schedule is blank!"
+        continue
+
```

Comment:
I assume you didn't mean to include these commits in this PR, so let's land the other PR first